### PR TITLE
[10.4] Document the occ maintenance:install database-connection-string option

### LINKED ISSUE #2127: Add support for Oracle connections string to support failover setups
## What Needs to be Documented?
New installation options for Oracle, see https://github.com/owncloud/core/pull/36489

## Where Does This Need To Be Documented?
Admin/Installation

## Why Should This Change Be Made? (Optional)
New option

## What Type Of Content Change Is This? (Optional)
- [x] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [ ] Bug Fix to Existing Content

## Which Manual Does This Relate To? (Optional)
- [x] Admin Manual
- [ ] Developer Manual
- [ ] User Manual
- [ ] Android
- [ ] iOS
- [ ] Branded Clients
- [ ] Desktop Client
- [ ] Other

@settermjd @DeepDiver1975

### PR DESCRIPTION
**Pull request by settermjd**
This fixes #2127 and relates to https://github.com/owncloud/core/pull/36489.

**Only to be merged into the 10.4 release.**

### CODE REVIEW

File: modules/admin_manual/pages/configuration/server/occ_command.adoc [hunk 1]
```diff
@@ -2782,24 +2782,49 @@ Usage:
...
-| `--database-host`          | Hostname of the database (default: `localhost`).
-| `--database-user`          | User name to connect to the database.
-| `--database-pass`          | Password of the database user.
-| `--database-table-prefix`  | Prefix for all tables (default: `oc_` ).
-| `--admin-user`             | Password of the admin account.
-| `--data-dir`               | Path to data directory (default: `/var/www/owncloud/data`).
+| `--database`                   
+| Supported database type (default: `sqlite`).
+| `--database-connection-string` 
+a| And Oracle-specific connection string.
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/server/occ_command.adoc]: I think this: `a| And Oracle` is mistyped and should be `|  A Oracle` 
If you look at how it is rendered, it makes it clear what is meant

Comment (settermjd) [reply modules/admin_manual/pages/configuration/server/occ_command.adoc]: No, it's fine how it is. The `a` prefix indicates that [the cell contains AsciiDoc content](https://asciidoctor.org/docs/user-manual/#cell).

Comment (settermjd) [reply modules/admin_manual/pages/configuration/server/occ_command.adoc]: > I think this: `a| And Oracle` is mistyped and should be `| A Oracle`
> If you look at how it is rendered, it makes it clear what is meant

That said, I see a grammatical mistake. Thanks for showing it to me.

Comment (settermjd) [reply modules/admin_manual/pages/configuration/server/occ_command.adoc]: Fixed.

**Pull request merged**